app/controllers/api/v1/packages_controller.rb

```diff
@@ -199,6 +199,9 @@ def undesignate_stockit_item
 
       def dispatch_stockit_item
         @orders_package = OrdersPackage.find_by(id: params[:package][:order_package_id])
+        order = @orders_package.order
+        render_order_status_error and return if order.prohibit_item_dispatch?
```

Comment:
Can't we directly use @orders_package.order.prohibit_item_dispatch? 🤔

Reply:
Hmmmmmmmmm